.gitignore

```diff
@@ -8,3 +8,6 @@ buildfiles
 /.gradle/
 /reports/
 /generated/
+*.iml
```

Comment:
Intellij still creates *.iml files within modules in some cases. Just had this issue last week with IDEA 2016.3.5 but dont know where it came from.